es-core/src/Sound.cpp

```diff
@@ -10,12 +10,17 @@ std::map< std::string, std::shared_ptr<Sound> > Sound::sMap;
 std::shared_ptr<Sound> Sound::get(const std::string& path)
 {
 	auto it = sMap.find(path);
-	if(it != sMap.cend())
+	if (it != sMap.cend())
```

Comment:
Style changes in separate commit is better.

Reply:
Reverted. Looking at the rest of this, some of the conditionals have spaces. But I agree that readability between diffs is easier when there are not too many changes especially when it just trivial.